behaviors/behaviors.py

```diff
@@ -4,6 +4,8 @@
 from django.db import models
 from django.utils import timezone
 
+from slugify import slugify
+
```

Comment:
Is there a reason to add a dependency on awesome-slugify and not use `django.utils.text import slugify`? we are just lower casing the slug anyways which is django utils slugify default behavior